spectacle.go

```diff
@@ -77,6 +76,7 @@ func main() {
 	gauge_messages.RegisterDocumenterServer(server, h)
 	fmt.Println(fmt.Sprintf("Listening on port:%d", l.Addr().(*net.TCPAddr).Port))
 	server.Serve(l)
+	os.Exit(0)
```

Comment:
do we need `os.Exit(0)`- isn't that the default?

Reply:
In this particular case, we may not need it. But when gauge sends a kill request it expects plugin to kill itself, this is to prevent orphan subprocess.
For Ex. in the case of python, gauge launches a shell process that launches the actual python runner.
If the plugin does not kill itself, the actual python process does not get kill (especially in windows) which leaves orphan processes. Cause gauge kills only shell process.

In this particular case, we might not need it since there is only one binary (golang) involved.

Reply:
Also earlier it was required cause we were breaking from a deferred function.
Now since it's in main, might not be required. Removed it.